mvp-spec/operator-requirements.md

```diff
@@ -30,13 +30,19 @@ clear how they will co-operate.
 We represent two typical workflows:
 
 - the first interaction with Prebid where a user visits a publisher's page, that triggers a CMP
-- a visit to a advertiser while the user already opted in to Prebid
+- a visit to an advertiser while the user already opted in to Prebid
+
+...with different configurations:
```

Comment:
NIT: create a proper sentence.

Reply:
I'm not sure to follow.
The sentence reads:
> We represent two typical workflows:
> [one of these workflows is] a visit to an advertiser while the user already opted in to Prebid